examples/file-feed/complete/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Anonymous+Pro|Source+Code+Pro" rel="stylesheet">
```

Comment:
Using default fonts will result in different look on different platforms. Using a consistent look (and a nicer-than-system-default) font is desirable. Is there a strong reason to **not** use 3rd party fonts?

@diasdavid yeah, I'll move the fonts to the CSS file.